debian/control

```diff
@@ -172,6 +177,7 @@ Architecture: linux-any
 Depends: ceph-base (= ${binary:Version}),
          python-flask,
          ${misc:Depends},
+         ${python:Depends},
```

Comment:
@smithfarm i don't think the monitor depends on python in any way. could you remove this line?